Zilon.Core/Zilon.Core/MapGenerators/RoomStyle/TownSectorScheme.cs

```diff
@@ -17,5 +17,6 @@ public class TownSectorScheme : SchemeBase, ISectorSubScheme
         public bool IsStart { get; }
         public int MinRegionMonsterCount { get; }
         public ISectorMapFactoryOptionsSubScheme MapGeneratorOptions { get; }
+        ISectorTransitionSubScheme[] ISectorSubScheme.TransSectorSids { get; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private property 'ISectorSubScheme.TransSectorSids'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5260137)